app/views/gardens/index.html.haml

```diff
@@ -22,43 +21,38 @@
 
   %table.table.table-striped
     %tr
+      %th 
       - unless @owner
         %th Owner
       %th Garden name
-      %th Description
-      %th Location
       %th Area
-      %th Active?
+      %th Location
       %th Plantings
-      %th
 
     - @gardens.each do |garden|
       %tr
+        
+        %td.status
```

Comment:
Any special reason why there's a class of `status` here? I think this and the classes `name` and `area` below might be a little too generic. We need to have conversation generally about how to organize CSS within Growstuff, so until then maybe better to remove these.